src/assertions.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+class Assertions:
+    @staticmethod
+    def assert_response_status(response: requests.Response, expected_status: int = 200):
+        """Verifica que el código de estado de la respuesta sea el esperado."""
+        assert response.status_code == expected_status, f"Expected status {expected_status}, got {response.status_code}"
+
+    @staticmethod
+    def assert_json_response(response: requests.Response):
+        """Verifica que la respuesta sea un JSON válido y no sea None."""
+        try:
+            response_data = response.json()
+        except ValueError:
+            raise AssertionError("Response is not a valid JSON")
+        assert response_data is not None, "Response JSON is None"
```

Comment:
este assert `assert response_data is not None, "Response JSON is None"` deberia ser otra funcion
ejemplo nombre de nueva funcion: `def assert_json_is_not_empty(response: requests.Response.json):`